### ISSUE TITLE
Support typedoc v0.27.6

### ISSUE THREAD

**Issue by taichunmin** [euberdeveloper/typedoc-plugin-ga#1]
An error will throw while using typedoc v0.27.6

```
The requested module 'typedoc' does not provide an export named 'default'
```

Full log:

```
[error] The plugin typedoc-plugin-ga could not be loaded
[error] file:///Users/user/git/chameleon-ultra.js/node_modules/typedoc-plugin-ga/bundled/lib/esm/index.esm.js:1
import t from"typedoc";function r(e){e.options.addDeclaration({name:"gaID",help:"Set the Google Analytics tracking ID and activate tracking code",type:t.ParameterType.String})}function o(e){e.renderer.hooks.on("body.end",()=>{let a=e.options.getValue("gaID");if(a){let n=`
       ^
SyntaxError: The requested module 'typedoc' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:177:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:260:5)
    at async onImport.tracePromise.__proto__ (node:internal/modules/esm/loader:543:26)
    at async loadPlugins (file:///Users/user/git/chameleon-ultra.js/node_modules/typedoc/dist/lib/utils/plugins.js:17:28)
    at async Application.bootstrapWithPlugins (file:///Users/user/git/chameleon-ultra.js/node_modules/typedoc/dist/lib/application.js:223:13)
    at async main (file:///Users/user/git/chameleon-ultra.js/node_modules/typedoc/dist/lib/cli.js:17:15)
```

**Comment by euberdeveloper** [comment]
I will soon check on your pul request,  thanks!

**Comment by euberdeveloper** [comment]
Fixed with [1.0.5](https://github.com/euberdeveloper/typedoc-plugin-ga/releases/tag/1.0.5). Please install a version equal or newer than 1.0.5 to make it work again
